llvm/lib/IR/AutoUpgrade.cpp

```diff
@@ -5773,7 +5773,7 @@ std::string llvm::UpgradeDataLayoutString(StringRef DL, StringRef TT) {
     if (!DL.contains("-p7") && !DL.starts_with("p7"))
       Res.append("-p7:160:256:256:32");
     if (!DL.contains("-p8") && !DL.starts_with("p8"))
-      Res.append("-p8:128:128");
+      Res.append("-p8:128:128:128:48");
```

Comment:
We might want a case that swaps `p8:128:128-` to `p8:128:128:128:48-`

Reply:
I just kept the current structure that doesn't touch pointer definitions that already exist - should we override those unconditionally? Or just the old variants?

Reply:
Fixed now.